cmd/add.go

```diff
@@ -36,13 +36,18 @@ func Add(subcomponent core.Component) (err error) {
 }
 
 var addCmd = &cobra.Command{
-	Use:   "add <component-name> --source <component-source> [--type component] [--method git] [--path .]",
-	Short: "Adds a subcomponent",
-	Long: `Adds a subcomponent.
-eg.
-$ fab add cloud-native --source https://github.com/timfpark/fabrikate-cloud-native
+	Use:   "'add <component-name> --source <component-source> [--type component] [--method git] [--path .]",
+	Short: "Adds a subcomponent to the current component (or the component specified by the passed path).",
+	Long: `Adds a subcomponent to the current component (or the component specified by the passed path).
+
+source: where the component lives (either a local path or remote http(s) endpoint)
+type: the type of component (component (default), helm, or static)
+method: method used to fetch the component (git (default))
+path: the path to the component that this subcomponent should be added to.
 
-where type defaults to "component" and method defaults to "git".
+example:
+
+$ fab add cloud-native --source https://github.com/timfpark/fabrikate-cloud-native
```

Comment:
can you change to point to the `/microsoft` cloud-native?

`fab add cloud-native --source https://github.com/Microsoft/fabrikate-definitions --path definitions/fabrikate-cloud-native `